hack/seccomp.pl

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/perl
```

Comment:
Also, of course you should commit generated code and not rely on generation tool. It is pretty hard to track bugs this way.